package.json

```
@@ -39,6 +39,7 @@
     "test:integration:generic": "E2E_APP_ID=react.vite.*,next.appRouter.withEmailCodes npm run test:integration:base -- --grep @generic",
     "test:integration:nextjs": "E2E_APP_ID=next.appRouter.* npm run test:integration:base -- --grep @nextjs",
     "test:integration:astro": "E2E_APP_ID=astro.* npm run test:integration:base -- --grep @astro",
+    "test:integration:expoWeb": "E2E_APP_ID=expo.expoWeb npm run test:integration:base -- --grep @expoWeb",
```

nit: same reason as above
```suggestion
    "test:integration:expo-web": "E2E_APP_ID=expo.expo-web npm run test:integration:base -- --grep @expo-web",
```